tubesize/src/popup.tsx

```
@@ -0,0 +1,128 @@
+            </div>
+            <div id="container">
+                <div className="cached-note">
+                    {videoData?.cached ? getCachedAgo(videoData.createdAt) : null}
+                </div>
+                {!videoData ? (
+                    <span className={message.type}> {message.message}</span>
+                ) : (
+                    videoData?.data?.videoFormats
+                        ?.map((item) => {
+                            return (
+                                <div className="format-item">
```

Biome (2.4.6)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)